.github/workflows/backend_actions.yml

```diff
@@ -0,0 +1,44 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Backend CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    
+    defaults:
+      run:
+        working-directory: ./backend
+        
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
Is there a reason to run it on all three versions? Otherwise seems like it works well... tests pass!

Reply:
This is something I saw that was common among other people's github actions. When I looked into it the answer seemed to be that node-version 12 is the most commonly used, but 12, 14, 16 are all current versions of node. It is checking that this project would work regardless of the version of node is being used.